tests/test_cli.py

```diff
@@ -17,7 +17,7 @@
     "cpp": ["clang", "-std=c++14"],
     "dart": ["dart", "compile", "exe"],
     "go": ["go", "tool", "compile"],
-    "julia": None,
+    "julia": ["julia"],
```

Comment:
maybe adding "--compiled-modules=yes" like below might reduce the cost of compiling and invoking separately.

The test runner can soon get a rewrite, requiring all new test cases have a "main", and making "INVOKER" the focus of the logic and then "COMPILER" is just an extra step needed for some langs.